features/karaf/openhab-tp/src/main/feature/feature.xml

```diff
@@ -154,7 +154,12 @@
 
   <feature name="openhab.tp-serial-rxtx" version="${project.version}">
     <capability>openhab.tp;feature=serial;impl=rxtx</capability>
-    <bundle>mvn:com.neuronrobotics/nrjavaserial/3.14.0</bundle>
+    <bundle>mvn:com.neuronrobotics/nrjavaserial/3.15.0</bundle>
+  </feature>
+
+  <feature name="openhab.tp-serial-rxtx-noliblockdev" version="${project.version}">
```

Comment:
Add a separate feature for the openHAB "no liblockdev" nrjavaserial.
The "important" point here is, that we also provide the specific capability, so our features know about the provided serial implementation.